pkg/catalog/catalog.go

```diff
@@ -2741,6 +2740,9 @@ func (c *Catalog) CopyEntry(ctx context.Context, srcRepository, srcRef, srcPath,
 		return nil, err
 	}
 
+	// Update creation date only after actual copy!!!
+	dstEntry.CreationDate = time.Now()
```

Comment:
I wonder - shouldn't we use the blockstore's mtime instead of `time.Now`? Is this even possible?

Reply:
Out of scope for this fix. Your suggestion is a valid question which has implications that we are currently  evaluating since it requires another call to the underlying storage